content/authors/adkris/_index.md

```diff
@@ -0,0 +1,77 @@
+---
+# Display name
+title: Aditya Krishna
+
+# Username (this should match the folder name)
+authors:
+- adkris
+
+weight: 11
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position
+role: Undergraduate Research Assistant
+
+# Organizations/Affiliations
+organizations:
+- name: Department of Electrical and Computer Engineering
+- name: Applied Physics Laboratory
+- name: University of Washington
+
+# Short bio (displayed in user profile at end of posts)
+bio: I am interested in applying artificial intelligence for environmental conservation
+
+interests:
+- Bioacoustics
+- Wildlife conservation
+- Artificial Intelligence 
+
+education:
+  courses:
+  - course: BE in Electrical Engineering (in progress)
+    institution: University of Washington
+    year: 2024 (expected)
+
+# Social/Academic Networking
+# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
+#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
+#   form "mailto:your-email@example.com" or "#contact" for contact widget.
+social:
+- icon: envelope
+  icon_pack: fas
+  link: 'mailto:adkris@uw.edu' # For a direct email link, use "mailto:test@example.org".
+- icon: github
+  icon_pack: fab
+  link: https://github.com/Adkris1002
+- icon: linkedin
+  icon_pack: fab
+  link: https://www.linkedin.com/in/adkris1002/
+- icon: paw
+  icon_pack: fas
+  link: https://wildlabs.net/users/adkris1002
+# Link to a PDF of your resume/CV from the About widget.
+# To enable, copy your resume/CV to `static/files/cv.pdf` and uncomment the lines below.
+# - icon: cv
+#   icon_pack: ai
+#   link: files/cv.pdf
+
+# Enter email to display Gravatar (if Gravatar enabled in Config)
+# email: ""
+
+# Organizational groups that you belong to (for People widget)
+#   Set this to `[]` or comment out if you are not using People widget.
+user_groups:
+- Researchers
+---
+
+I am a student at the University of Washington majoring in Electrical Engineering with a concentration in Digital Signal Processing. I will be graduating around the Spring of 2024 and look forward to pursuing graduate studies after receiving my bachelor's degree. 
```

Comment:
suggested change:
"I will be graduating around the Spring of 2024 ..." --> "I am expected to graduate ..."
and corresponding grammar change in the second part of the sentence.